client/device/cloudftd/cloudftdonboarding/create.go

```diff
@@ -166,6 +167,7 @@ func Create(ctx context.Context, client http.Client, createInp CreateInput) (*Cr
 				Build(),
 		),
 		retry.NewOptionsBuilder().
+			Message("waiting for Cloud FTD onboarding to be done").
```

Comment:
Waiting for FTD to be onboarded to CDO...

Reply:
done 👍